src/models/auth.cr

```diff
@@ -1,9 +1,17 @@
 class Auth
+  @signing_key : String
+
   def initialize(@scopes : String)
-    @file_path = File.expand_path(ENV["GOOGLE_AUTH_FILE"])
+    @signing_key = read_base64_key(ENV["GOOGLE_PRIVATE_KEY"])
+    @issuer = ENV["GOOGLE_CLIENT_EMAIL"]
```

Comment:
Potential for key error here.

Reply:
what do you mean, if we dont load env, won't it just throw a missing env anyways?